src/Components/HideIcon.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome'
```

Comment:
The project is based on Material UI which already comes with its own React Icon component and its own set of icons (https://mui.com/material-ui/icons/).

Adding Fort Awesome as a new dependency solely for icons is redundant.

(I am aware that this dependency was introduced in a different commit)

/cc @pablo-mayrgundter